tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -176,33 +178,50 @@ def test_plasma_properties(self, plasma, tardis_ref_data, config, attr):
             key = os.path.join(config.plasma.save_path, "plasma", attr)
             expected = tardis_ref_data[key]
             pdt.assert_almost_equal(actual, expected)
+            if isinstance(actual, (pd.DataFrame, pd.Series)):
+                assert snapshot_pd == actual
+            else:
+                assert snapshot_np == actual
         else:
             warnings.warn(f'Property "{attr}" not found')
 
-    def test_levels(self, plasma, tardis_ref_data, config):
+    def test_levels(
+        self, plasma, tardis_ref_data, config, snapshot_pd, snapshot_np
+    ):
         actual = pd.DataFrame(plasma.levels)
         key = os.path.join(config.plasma.save_path, "plasma", "levels")
         expected = tardis_ref_data[key]
         pdt.assert_almost_equal(actual, expected)
```

Comment:
Should this assert statement still be included? Do the next four lines do the same thing?

Reply:
Yes, this was from the old code and would be removed!